app/importers/file_importers/educators_importer.rb

```diff
@@ -88,7 +88,7 @@ def import_row(row)
     end
 
     # Update homeroom
-    homeroom = Homeroom.find_by_name(row[:homeroom])
+    homeroom = Homeroom.find_by(name: row[:homeroom], school: educator.school)
```

Comment:
@alexsoble I'm not sure if this is helpful, apologies if not, but this might be related to https://github.com/studentinsights/studentinsights/issues/1880#issuecomment-408219241, this will find homerooms based on the *educator* school.  But they'll be created later based on the *student* school.  There's a bunch of edge cases then if those don't match.

I wonder if there's a way to get a primary key for homerooms, or to create a composite one from (`school_local_id`, `homeroom_name`) and if that's effectively unique or not.

Reply:
@kevinrobinson Yes, this is helpful! 

I like the idea of getting primary key for homerooms, but I know that the network/IT admins for Somerville and New Bedford are going to be busy with start of school right now, so my preference would be for a solution that doesn't involve asking them to update scripts. 

I ran queries on Somerville and New Bedford and found that all homerooms were unique on (`school_local_id`, `homeroom_name`). As a next step I added an index in 77d29d86a742399f4eaeb367d4a37c022f567a9a with `unique: true` to enforce that uniqueness at the database level. 

I'll also move homeroom creation from the students importer to educators importer, so that the create/update paths are in the same place and all the students importer does with homerooms is read them and assign students to them.

Reply:
@alexsoble those sound like big changes, and great simplifications!

Reply:
@kevinrobinson Agreed! I'm going to split this off into two PRs.